saw-script.cabal

```diff
@@ -96,6 +96,7 @@ library
     , flexdis86
     , elf-edit
     , reflection
+    , prettyprinter
```

Comment:
Is this a duplicate? `prettyprinter` should already appear above this.

Reply:
Yep. Fixed.